modules/manage/pages/cluster-maintenance/disk-utilization.adoc

```diff
@@ -56,18 +58,28 @@ Default - `null`
 | `retention.local.target.bytes`  +
 No default
 
+| Segment lifetime
+| `log_segment_ms`  +
+No default
+| `segment.ms`  +
+No default
+
 | Segment size
 | `log_segment_size`  +
 Default - `1073741824`
 | `segment.bytes`  +
 No default
 |===
 
-Data expires from object storage following `retention.ms` or `retention.bytes`. For example, if `retention.bytes` is set to 10 GiB, then every partition in the topic has a limit of 10 GiB storage in the cloud. When `retention.bytes` is exceeded by data in object storage, the data in object storage is trimmed. With Tiered Storage enabled, data expires from local storage following `retention.local.target.ms` or `retention.local.target.bytes`.
+Data expires from object storage following both `retention.ms` and `retention.bytes`. For example, if `retention.bytes` is set to 10 GiB, then every partition in the topic has a limit of 10 GiB storage. When `retention.bytes` is exceeded by data in object storage, the data in object storage is trimmed, even if `retention.ms` is not yet exceeded. With Tiered Storage enabled, data expires from local storage following `retention.local.target.ms` or `retention.local.target.bytes`.
```

Comment:
Might also add a sentence to the end of this like:

> Refer to the Space Management section to see how dynamic storage management can impact local storage cleanup.

Specifically because setting retention_local_strict=false means we can ignore retention.local.target.* settings when the system is under disk pressure, allowing for earlier removal of segmetns.

Reply:
I like this idea, added a sentence with the reference!